mrs_msgs/Reference[] ground_waypoint_vector